lib/main-menu.js

```diff
@@ -153,6 +153,19 @@ var view = {
   label: 'View',
   submenu: [
     {
+      label: 'Toggle Full Screen',
+      accelerator: (() => {
+        if (process.platform === 'darwin') return 'Ctrl+Command+F'
+        return 'F11'
+      })(),
+      click: (item, focusedWindow) => {
```

Comment:
Actually it is used and also needed, because when building the menu-item `Toggle Full Screen`, the `click` property is expected to be a callback with two parameters: 
- `item`, represents the `Toggle Full Screen` object inside the menu, and clicking it triggers the execution of the below of it defined code 
- and `focusedWindow`, which works as the bridge between the menu and the main app process allowing us to pass through the message inside of the [send() function](https://github.com/BoostIO/Boostnote/pull/491/files#diff-c4d0b50cf156e17e35d992365a6fb80aR161)

I know that it is a bit confusing, but this is how electron operates, and without defining it inside of the `click` property the code will result in an error. It is displayed more on electon's official [menu-example](https://github.com/electron/electron-docs/blob/master/test/fixtures/menu.md#menu), where you can spot it after scrolling down a bit : )

I hope my explanation was somehow helpful! 😄